srcpkgs/mu4e/template

```diff
@@ -11,7 +11,7 @@ maintainer="Benjamin Slade <slade@lambda-y.net>"
 license="GPL-3.0-or-later"
 homepage="https://www.djcbsoftware.nl/code/mu/"
```

Comment:
```suggestion
homepage="https://www.djcbsoftware.nl/code/mu/"
changelog="https://github.com/djcb/mu/raw/master/NEWS.org"
```

Reply:
I intentionally didn't add this as it doesn't reflect changes of minor versions. The changelog seems to be partially in `News.org` and in the description of the releases on github. Still want me to add it?

Reply:
i think it's still good to add

Reply:
done.